llm/gemma/README.md

```diff
@@ -0,0 +1,62 @@
+# Serve Your Gemma on Any Cloud
+
+Google released [Gemma](https://blog.google/technology/developers/gemma-open-models/) and has made a big wave in the AI community.
+It opens the opportunity for the open-source community to serve and finetune private Gemini.
+
+## Serve Gemma on any Cloud
+
+Serving Gemma on any cloud is easy with SkyPilot. We can find the [serve.yaml](serve.yaml) in this directory, which can be used to start hosting the model on any cloud with a single command.
+
+### Prerequsite
+
+1. Apply for the access to the Gemma model
+
+Go to the [application page](https://huggingface.co/google/gemma-7b) and apply for the access to the model weights.
+
+
+2. Get the access token from huggingface
+
+Generate a read-only access token on huggingface [here](https://huggingface.co/settings/token), and make sure your huggingface account can access the Gemma models [here](https://huggingface.co/google/gemma-7b).
+
+
+### Host on a Single Instance
+
+We can host the model with a single instance:
+```bash
+sky launch -c gemma serve.yaml --env HF_TOKEN="xxx"
```

Comment:
nit: How about `HF_TOKEN=xxx sky launch ...`?

Reply:
(same for sky serve up below?)

Reply:
Specifying the `HF_TOKEN` as a env directly won't work in our `sky launch` : )

Reply:
I meant `HF_TOKEN=xxx sky launch --env HF_TOKEN ...`.